content/asp.net/getting-started/views.md

```diff
@@ -1,20 +1,121 @@
+
+Finally, the `_Layout.cshtml` is used as a template for rendering your web pages and is located in the `Views\Shared` folder of your project.  Layouts allow you to define mandatory and optional sections of what will become your rendered page that can be later filled in by your views. A default layout is created in the project template complete with common styles and page head information, a navigation bar, a footer, a section stubbed in for the body of the page as well as an optional section to add scripts to the final output in your views.  You can have as many layouts as are needed, perhaps to enable specific functionality for a subset of users or to provide a different experience in an administration area in your site. 
+
+## Using Razor View Engine Features
+
+When you employ these features in unison it is important to remember a few of the rules that govern their use by the view engine. Razor looks for direction on layouts and imports at each level in the folder structure. `Using` statements that bring in a namespace at the root level will be available in all subfolders. Likewise, tag helpers that are brought in at a subfolder level will not be available to other "sibling" folders unless you explicitly declare them there as well. Finally, `ViewImports` are additive and allow you to add or remove namespaces and tag helpers, but declaring imports at a subfolder level will not override or clear out previously imported components.
+
+In the project template we see a very basic set of statements in our `ViewImports.cshtml` document that expose the root project namespace and add in the Microsoft tag helpers library. This is the perfect place to add additional tag helpers that you may write yourself or pull in from other third-party libraries. Defining them here makes them available throughout all views in the project (due to the inheritance model used by Razor).
+
+``` csharp
+@using MvcBasics
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+```
+
+The `ViewStart` is also pretty straightforward in the template, but its use helps us to understand a little better what is going on. You can think of this code as the code that will be appended to the start of each class that is generated when our views are compiled. In this case, the layout that will be used for all views is assigned to the Layout property of the class. This can be overridden in views if needed. This would also be a good place to evaluate if a different layout should be used for the properties associated with the request, namely a user claim or perhaps the host name.
```

Comment:
"This would also be a good place to evaluate if a different layout should be used for the properties associated with the request, namely a user claim or perhaps the host name."
Consider:
"This would also be a good place to customize the view's layout, perhaps based on properties of the request, such as a user claim or the host name."